files/jira.rb

```diff
@@ -32,9 +49,18 @@ def create_issue(summary, full_description, project)
             ]
           }
         }
-        issue.save(issue_json)
-        url = get_options[:site] + '/browse/' + issue.key
-        puts "Created issue #{issue.key} at #{url}"
+        # FIXME - would this work
+        components = []
+        if components.size?
+          issue_json['fields']['components'] = components
```

Comment:
This is where I put some real code, having added functionality for jira_components to come from the config :)